.github/workflows/release.yml

```diff
@@ -76,15 +76,17 @@ jobs:
         uses: actions/checkout@v2
       - name: Setup editorconfig checker
         run: |
-          wget https://github.com/editorconfig-checker/editorconfig-checker/releases/download/2.1.0/ec-linux-amd64.tar.gz
-          tar xvf ec-linux-amd64.tar.gz
+          ls /tmp/bin/ec-linux-amd64 || \
+          cd /tmp && \
+          wget https://github.com/editorconfig-checker/editorconfig-checker/releases/download/2.1.0/ec-linux-amd64.tar.gz && \
+          tar xvf ec-linux-amd64.tar.gz && \
           chmod +x bin/ec-linux-amd64
       - name: Check files
-        run: bin/ec-linux-amd64
+        run: /tmp/bin/ec-linux-amd64
 
   check-prettier:
     name: "Check with Prettier"
-    runs-on: ubuntu-latest
+    runs-on: self-hosted
```

Comment:
I wonder if it makes sense to keep these fast ones on github-hosted?

Reply:
Thought about it but I didn't see much value